plugins/woocommerce/src/StoreApi/Schemas/V1/AbstractAddressSchema.php

```diff
@@ -236,23 +236,24 @@ function( $key ) use ( $additional_fields_keys ) {
 			ARRAY_FILTER_USE_KEY
 		);
 
-		$schema = array();
+		$schema = [];
 		foreach ( $address_fields as $key => $field ) {
-			$field_schema = array(
+			$field_schema = [
 				'description' => $field['label'],
 				'type'        => 'string',
 				'context'     => [ 'view', 'edit' ],
 				'required'    => true,
-			);
+			];
 
 			if ( 'select' === $field['type'] ) {
-				$field_schema['type'] = 'string';
 				$field_schema['enum'] = array_map(
 					function( $option ) {
 						return $option['value'];
 					},
 					$field['options']
 				);
+			} elseif ( 'checkbox' === $field['type'] ) {
```

Comment:
Slight nitpick but would improve readability.

Does this need to be an `elseif` can't it just be an independent if?